lm75bd/lm75bd.c

```diff
@@ -26,8 +27,31 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
+  error_code_t errCode;
+
+  /* Selecting temperature register */
+  uint8_t sendData = 0;
+  errCode = i2cSendTo(devAddr, &sendData, 1);
+  if (errCode != ERR_CODE_SUCCESS) return errCode;
+
+  /* Receive temp reading */
+  uint8_t buff[2] = {0, 0};
+  errCode = i2cReceiveFrom(devAddr, buff, 2);
+  if (errCode != ERR_CODE_SUCCESS) return errCode;
```

Comment:
RETURN_IF_ERROR_CODE can be used to log and return the error code.
```suggestion
  /* Selecting temperature register */
  uint8_t sendData = 0;
  RETURN_IF_ERROR_CODE(i2cSendTo(devAddr, &sendData, 1));

  /* Receive temp reading */
  uint8_t buff[2] = {0, 0};
  RETURN_IF_ERROR_CODE(i2cReceiveFrom(devAddr, buff, 2));
```